### DISCUSSION TITLE
OpenParEM released

### DISCUSSION THREAD

**Discussion by M8kmyday** [mfem/mfem#4512] [Show and tell]
I just wanted to let the MFEM team know that I have released OpenParEM (https://openparem.org), which is of course, heavily dependent on MFEM.  In short, it is a full-wave frequency-dependent 3D electromagnetic simulator producing S-parameters between wave ports plus fields.  Thanks for all the help over the 3 years (full time on my part) to get this project to its first releasable state.  I do plan to continue working on it, adding capability, bug fixes, etc.

**Comment by tzanio** [comment]
Congratulations @M8kmyday !
